### ISSUE TITLE
Integrate realtime drawing

### ISSUE THREAD

**Issue by bigfish24** [realm-demos/realm-teamwork-MR#22]
To make the app demonstrate more realtime or reactive experiences we could add the ability to take a photo of something to create a task (similar to a service management style app), where the photo could then be marked up and synced in realtime. This could utilize the same drawing code in https://github.com/realm-demos/realm-draw. Example would be something like: http://www.fieldwire.com/

**Comment by dhmspector** [comment]
Yes!  Max & I had been talking about exactly that app as an example of what we should create. 

After consulting with @TimOliver it looks like our currently available code is probably a pessimal solution for this at best. There are probably several open source drawing kits we can fork to make this possible faster that improving the older code base.   Will investigate & write up the most likely candidates in the next few days.
